lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -86,7 +86,9 @@ class PolicyElement < ::ActiveRecord::Base
 
       # needs unique_identifier, policy_machine_uuid, type, extra_attributes columns
       has_many :assignments, foreign_key: :parent_id, dependent: :destroy
+      has_many :filial_ties, class_name: 'Assignment', foreign_key: :child_id
```

Comment:
I was just thinking about that.  This would account for the orphaned assignments we saw.  Adding.